### ISSUE TITLE
Does not work on Kubernetes 1.19

### ISSUE THREAD

**Issue by realshuting** [stackrox/admission-controller-webhook-demo#8]
I went through the setup, when creating a pod, got this error, and the pod did not get mutated:

```
2020/10/02 00:29:59 http: TLS handshake error from 10.244.0.1:16521: remote error: tls: bad certificate
```

**Comment by sergiocharpineljr** [comment]
I can confirm. The error is:

certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0\n

The deploy.sh should be changed to use altnames.

**Comment by misberner** [comment]
This was fixed by #11
